pkg/cfn/manager/api.go

```diff
@@ -220,3 +249,61 @@ func (c *StackCollection) DescribeStackEvents(i *Stack) ([]*cloudformation.Stack
 	}
 	return resp.StackEvents, nil
 }
+
+func (c *StackCollection) doCreateChangesetRequest(i *Stack, action string, description string, templateBody []byte,
+	parameters map[string]string, withIAM bool) (string, error) {
+
+	changesetName := fmt.Sprintf("eksctl-%s-%d", action, time.Now().Unix())
```

Comment:
Is there max length to this? It doesn't appear to get namespaced:

```
arn:aws:cloudformation:eu-west-1:123456789:changeSet/eksctl-scale-nodegroup-1539703695/0d6c1284-4d0d-44d8-9827-d1eaff527d4a
```

It'd be better if prefixed it with stack name, e.g. `eksctl-attractive-wardrobe-1539702643-nodegroup-0-scale-<timestamp>`.

Reply:
Sure, we can do that

Reply:
I'll add and push a new change tomorrow